crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -290,7 +296,6 @@ where
         ldk_config: UserConfig,
         settings: LnDlcNodeSettings,
         oracle_client: P2PDOracleClient,
-        event_sender: Option<watch::Sender<Option<Event>>>,
         read_scorer: SC,
     ) -> Result<Self>
     where
```

Comment:
hm, I'll double-check that! thanks for spotting it.

Reply:
indeed, a few tasks were interspersed with building the entities. Moved them all!

also, left a comment that potentially it would be good to keep handles for all of them? This might be one of the reasons our app doesn't shutdown / hot reload nicely....